recipes/igdiscover/meta.yaml

```diff
@@ -0,0 +1,47 @@
+about:
+  home: https://igdiscover.readthedocs.io/
+  license: 'MIT'
+  summary: 'Analyze antibody repertoires and discover new V genes'
+package:
+  name: igdiscover
+  version: "0.3"
+source:
+  fn: igdiscover-0.3.tar.gz
+  url: https://pypi.python.org/packages/ba/ff/bc398601c92d05bf502bcd3f49897b20cd3a0fe41cde0f17c8f073222ad4/igdiscover-0.3.tar.gz#md5=132e771db3e168018a613c0900b560d9
+  md5: 132e771db3e168018a613c0900b560d9
+requirements:
+  run:
+    - python >=3.3
```

Comment:
Constraining the Python version here usually results in conda-build not pinning the Python version in the package. This is clearly a shortcoming in conda, but until it is solved, we have to omit the constraint here and rely solely on the skip option below.